kube/services/arborist/arborist-deploy.yaml

```diff
@@ -86,6 +94,16 @@ spec:
         env:
           - name: PYTHONPATH
             value: /var/www/fence
+          - name: ARBORIST_BUCKET_NAME
+            valueFrom:
+              configMapKeyRef:
+                name: global
```

Comment:
Do not add new variables to the `global` configmap - use the gitops `manifest.json` and the `manifest-global` configmpap! :tada:

Reply:
Fixed in cb8b9075f234dbda5264e82022b47f6581399aaf. @rudyardrichter  I also moved `sync_from_dbgap` to `manifest-global`